Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.py

```diff
@@ -0,0 +1,92 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+#   from typing import Dict
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def __init__(self, server_url: str, client_id: str, client_secret: str, proxy: bool, verify: bool):
+        super().__init__(base_url=server_url, proxy=proxy, verify=verify)
+        self._client_id = client_id
+        self._client_secret = client_secret
+        self._token = self._generate_token()
+        self._headers = {'Authorization': self._token, 'Content-Type': 'application/json'}
+
+    def _generate_token(self) -> str:
+        body = {
+            "client_id": self._client_id,
+            "client_secret": self._client_secret,
+            "grant_type": "client_credentials",
+            "provider": "thy-one"
+        }
+
+        headers = {
+            'Content-Type': 'application/x-www-form-urlencoded'
+        }
+        return "Bearer " + (self._http_request("POST", "/v1/token", headers=headers, data=body)).get('accessToken')
+
+    def getSecret(self, name: str) -> str:
+        return self._http_request("GET", url_suffix="/v1/secrets/" + str(name))
+
+
+def dsv_secret_get_command(client, name: str = ''):
+    secret = client.getSecret(name)
+
+    return CommandResults(
+        readable_output=f"{secret}",
+        outputs_prefix="DSV.Secret",
+        outputs_key_field="secret",
+        raw_response=secret,
+        outputs=secret
+    )
+
+
+def test_module(client) -> str:
+    if client._token == '':
+        raise Exception('Failed to get authorization token. Check you credential and access to Secret Server.')
+
+    return 'ok'
+
+
+def main():
+    client_id = demisto.params().get('client_id')
+    client_secret = demisto.params().get('client_secret')
+
+    # get the service API url
+    url = demisto.params().get('url')
+    proxy = demisto.params().get('proxy', False)
+    verify = not demisto.params().get('insecure', False)
+#    credential_objects = demisto.params().get('credentialobjects')
+
+    LOG(f'Command being called is {demisto.command()}')
+
+    thycotic_commands = {
+        'dsv-secret-get': dsv_secret_get_command
```

Comment:
1. Can you explain the use case of this command, please?
2. Can you add some information and details on the differences between Thycotic Secret Server and DSV?
3. Any chance you could elaborate on why this integration does not fetch credentials?